libraries/c_sdk/standard/mqtt/src/iot_mqtt_mutex_wrapper.c

```diff
@@ -81,13 +91,13 @@ bool IotMutex_CreateRecursiveMutex( StaticSemaphore_t * pMutex )
     return status;
 }
 
-bool IotMutex_Take( StaticSemaphore_t * pMutex )
+bool IotMutex_Take( SemaphoreHandle_t * pMutex )
```

Comment:
Why not just have `SemaphoreHandle_t mutex`?

Reply:
It can be Semaphore Handle itself. I will do it as a follow up PR.